validator/src/main.rs

```diff
@@ -223,13 +227,6 @@ fn initialize_ledger_path(
     Ok(genesis_blockhash)
 }
 
-// Return an error if a keypair file cannot be parsed.
-fn is_keypair(string: String) -> Result<(), String> {
```

Comment:
This is small win resulting from [this](https://github.com/solana-labs/solana/pull/6790/files#r343502742).